changelog.md

```diff
@@ -1,3 +1,252 @@
+
+ <a name="6.0.0-beta.0"></a>
+# 6.0.0-beta.0 (2020-03-11)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v5.6.0...v6.0.0-beta.0)
+
+(A proper thanks for our new contributors will be in the final 6.0 release notes.)
+
+This beta release will not ship in Chrome DevTools, but we'll ship the final 6.0 release.
```

Comment:
Do we also want to call out that the beta won't be shipped to the PSI front-end for now?